README.md

```diff
@@ -89,3 +91,44 @@ const appAuth = new LOAuth({
   }
 });
 ```
+
+## Using API-based Authentication
+
+```ts
+import { APIBasedAuth } from '@lifeomic/app-tools';
+
+// Setup
+const auth = new APIBasedAuth({
+  clientId: '<clientId>',
+  storage: {
+    // Provide a mechanism for storing + retrieving tokens.
```

Comment:
Is there a default for this? Maybe worth filling out example with local storage or maybe a sync storage